src/classes/Foo.cls

```diff
@@ -0,0 +1,16 @@
+public class Foo {
```

Comment:
Class `Foo` does not enforce sharing correctly. Please change to `Inherited Sharing`.